integration_tests/test_aggregator_settlement.py

```diff
@@ -0,0 +1,159 @@
+# pylint: disable=too-many-instance-attributes, broad-except
+import logging
+import traceback
+import random
+import json
+
+from gsy_framework.constants_limits import DATE_TIME_FORMAT
+from pendulum import from_format
+
+from gsy_e_sdk.clients.redis_asset_client import RedisAssetClient
+from integration_tests.test_aggregator_base import TestAggregatorBase
+
+
+class SettlementAggregator(TestAggregatorBase):
+    """Aggregator class to test the connections to settlement
+     markets:
+     Done by asserting whether bids/offers are successfully
+     posted on these markets."""
+
+    def __init__(self, *args, **kwargs):
+
+        self.base_forecast_load = 10
+        self.base_measurement_load = self._calculate_random_deviated_energy(
+            self.base_forecast_load, 5)
+        self.base_forecast_pv = 10
+        self.base_measurement_pv = self._calculate_random_deviated_energy(
+            self.base_forecast_pv, 5)
+
+        self._has_tested_settlement_posts = False
+        self._has_tested_energy_exposure = False
+        super().__init__(*args, **kwargs)
+
+    def _setup(self):
+        load_client = RedisAssetClient("forecast-measurement-load", pubsub_thread=self.pubsub)
+        load_client.select_aggregator(self.aggregator_uuid)
+        self.load_uuid = load_client.area_uuid
+
+        pv_client = RedisAssetClient("forecast-measurement-pv", pubsub_thread=self.pubsub)
+        pv_client.select_aggregator(self.aggregator_uuid)
+        self.pv_uuid = pv_client.area_uuid
+
+    @staticmethod
+    def _calculate_random_deviated_energy(energy, off_percentage):
+        delta = energy * off_percentage / 100
+        sign = 1 if random.random() < 0.5 else -1
+        return energy + sign * delta
+
+    @staticmethod
+    def _can_place_forecast_measurements(area_dict):
+        return "area_name" in area_dict and ("forecast-measurement" in area_dict["area_name"])
+
+    @staticmethod
+    def _next_market_slot(market_info):
+        return from_format(market_info["market_slot"], DATE_TIME_FORMAT) \
+            .add(minutes=60).format(DATE_TIME_FORMAT)
```

Comment:
Use `()` instead of `\` as per [PEP 8](https://realpython.com/python-pep8/#indentation-following-line-breaks) recommendation